### ISSUE TITLE
Clicking History entries in upgraded app causes "space does not exist" errors

### ISSUE THREAD

**Issue by philrz** [brimsec/brim#813]
In the attached video, I start out running GA Brim tagged `v0.9.1`. I perform a couple queries to populate my History panel. Then I close the app and upgrade to a release artifact based on Brim commit `dc488b1`. My History entries now show the yellow exclamation point warning, and when I click on any of them, I get a "space does not exist" error.

[newbug.mp4.zip](https://github.com/brimsec/brim/files/4683170/newbug.mp4.zip)

I sniffed around in Wireshark to see if it's likely a frontend or backend bug. Here's the initial exchange in `v0.9.1`:

![image.png](https://images.zenhubusercontent.com/5e40bbae92d04e424a90e56f/e253e062-0fe4-439a-8db5-5d53784db575)

Then in the newer version, I see a lot more info was returned in the initial `GET /space`, but when the Search is issued, there's no `space` field like there was in `v0.9.1`

![image.png](https://images.zenhubusercontent.com/5e40bbae92d04e424a90e56f/a751e449-3239-4fda-8edc-8115a2441020)

**Comment by jameskerr** [comment]
This seems to me like a front end error related to the changes to the Space API. Thanks @philrz

**Comment by philrz** [comment]
Verified in Brim commit `9373ee8`. In the attached video, I've started with the repro steps in `v0.9.1` as shown in the prior video, but now upgrade to a new release artifact based on commit `9373ee8`. As shown, now my History entries no longer have the yellow exclamation mark icons, and I can click on them and re-run the queries.

[Verify.mp4.zip](https://github.com/brimsec/brim/files/4703475/Verify.mp4.zip)

Thanks @mason-fish!
